src/Events/QueryCreatedEvent.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/*
+
+This file was created by developers working at BitBag
+
+Do you need more information about us and what we do? Visit our   website!
+
+We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+declare(strict_types=1);
+
+namespace BitBag\SyliusCrossSellingPlugin\Events;
+
+use Elastica\Query\AbstractQuery;
+use Symfony\Contracts\EventDispatcher\Event;
+
+final class QueryCreatedEvent extends Event implements QueryCreatedEventInterface
```

Comment:
Isn't this logic already implemented within BitBag's ElasticPlugin? If so, then why we're not using that one in here?